pageserver/src/layered_repository.rs

```diff
@@ -1649,7 +1649,7 @@ impl LayeredTimeline {
                     continue;
                 }
 
-                info!(
```

Comment:
I suggest removing the `continue` above, since it was added due to the same reason: https://github.com/zenithdb/zenith/pull/1427